src/sqlancer/stonedb/gen/StoneDBTableCreateGenerator.java

```diff
@@ -153,6 +153,8 @@ private void addCommonErrors(ExpectedErrors list) {
         list.add("A BLOB field is not allowed in partition function");
```

Comment:
Did you check whether the other expected errors are actually needed?

Reply:
Will gradually add each expected errors by each commit

Reply:
My comment was about the existing errors that you have added without feedback by the DBMS. Are they actually needed? I'd suggest to remove them and only re-add those that are needed.

Reply:
Oh, got it. Will remove the errors added before today's test and only add them back when I got one.